core/src/main/java/org/projectnessie/cel/interpreter/AttributeFactory.java

```diff
@@ -1215,6 +1217,12 @@ static Val refResolve(TypeAdapter adapter, Val idx, Object obj) {
       }
       return elem;
     }
+    if (celVal instanceof Lister) {
```

Comment:
Isn't this `if` superfluous, because `Lister extends Indexer`?

Reply:
You are right. Let me remove this redundant check